content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity contains GB and GiB. 
```

Comment:
How is the content in `short_descripton` related? I don't think this is how we'd like to see it.

Reply:
okay, can you please suggest me to add something here?